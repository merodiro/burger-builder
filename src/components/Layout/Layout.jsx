import React, { Fragment } from 'react'

import classes from './Layout.module.css'
import Toolbar from '../Navigation/Toolbar/Toolbar'
import SideDrawer from '../Navigation/SideDrawer/SideDrawer'

const Layout = ({ children }) => (
  <Fragment>
    <Toolbar />
    <SideDrawer />
    <main className={classes.Content}>{children}</main>
  </Fragment>
)

export default Layout
