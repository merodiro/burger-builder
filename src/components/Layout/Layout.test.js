import React from 'react'
import { shallow } from 'enzyme'

import Layout from './Layout'

describe('<Layout>', () => {
  it('renders the same output', () => {
    const component = shallow(<Layout />)

    expect(component).toMatchSnapshot()
  })

  it('renders with children', () => {
    const component = shallow(<Layout>Hello world</Layout>)

    expect(component).toMatchSnapshot()
  })
})
