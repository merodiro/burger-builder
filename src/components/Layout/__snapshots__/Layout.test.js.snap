// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout> renders the same output 1`] = `
<React.Fragment>
  <Toolbar
    clicked={[Function]}
  />
  <SideDrawer
    closed={[Function]}
    open={false}
  />
  <main
    className="Content"
  />
</React.Fragment>
`;

exports[`<Layout> renders with children 1`] = `
<React.Fragment>
  <Toolbar
    clicked={[Function]}
  />
  <SideDrawer
    closed={[Function]}
    open={false}
  />
  <main
    className="Content"
  >
    Hello world
  </main>
</React.Fragment>
`;
